src/gui/tray/Window.qml

```diff
@@ -564,6 +667,13 @@ Window {
                     ToolTip.delay: 1000
                     ToolTip.text: qsTr("Open share dialog")
                     onClicked: Systray.openShareDialog(displayPath,absolutePath)
+
+                    Accessible.role: Accessible.Button
+                    Accessible.name: qsTr("Share button")
+                    Accessible.description: qsTr("Opens share window for the corresponding sync object")

```

Comment:
Instead, name should be "Share `file/foldername`" and no description is needed. :)